### PR TITLE
doc: add vision and mission

### PR DESCRIPTION
**Pull request by MylesBorins**
The mission / values statement drafted in the board's merger
subcommittee were used as a baseline for decision making and
direction. This commit captures these in the README of the repo

### CODE REVIEW

**Comment by bnb** [comment]
Not sure if this conflicts with what we agreed to PR with the rest of the group in https://github.com/nodejs/bootstrap/pull/4

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
 Foundation Bootstrap Team
+
+## What's in a name?
+
+As we do not yet have a name for a merged entity we shall refer to merged entity in this document
+as $FOUNDATION
+
+## Mission
+
+$FOUNDATION envisions a future where JavaScript is a sustainable platform for implementing best in
```
Comment (ChALkeR) [inline README.md]: Does this correspond to what was discussed at the summit?
«envisions a future where …» part seems pretty controversial to me.

Comment (jasnell) [reply README.md]: Not entirely. In the afternoon session we worked through an exercise of creating several possible mission statements. We concluded that each of those should be posted as work-in-progress statements and that we would coallesce into one over time. I think those statements should be added here with clear language indicating work in progress.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,22 @@
...
+class experiences in production across all types of workloads.  The $FOUNDATION is committed to
```
Comment (yorkie) [inline README.md]:
```suggestion
class experiences in production across all types of workloads. The $FOUNDATION is committed to
```
This looks like a unneeded space :)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,22 @@
 # bootstrap
 Foundation Bootstrap Team
+
+## What's in a name?
+
+As we do not yet have a name for a merged entity we shall refer to merged entity in this document
+as $FOUNDATION
```
Comment (yorkie) [inline README.md]:
```suggestion
as $FOUNDATION.
```

**Comment by amiller-gh** [comment]
@MylesBorins I've included these two drafts as possible mission / vision statement drafts in `FOUNDATION.md` in #7. If pulled in, this PR can be closed as a historical document.

**Pull request closed without merging**